guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -1,55 +1,247 @@
+Otherwise it maps to *OK*.
+* *Error*: This indicates an error during configuration.
+For example, a run failed to install a package.
+* *Out of sync*: A configuration report was not received within the expected interval, based on the `outofsync_interval` setting.
+Reports are identified by an origin and can have different intervals based upon it.
 
-The *configuration* sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
+Build::
+This sub-status is only relevant for managed hosts, hosts registered through global registration, and when {Project} runs in unattended mode.
++
+Possible values:
+// TODO: numbers
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Pending | OK | 0
+| Built | OK | 0
+|===
 
-To view the sub-status for a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and click the host whose full status you want to inspect.
-You can also view substatus information in the hover help for each host.
 
-In the *Properties* table of the host details' page, you can view both the global host status and all sub-statuses.
+ifdef::satellite,orcharhino[]
+Execution::
+Status of the last completed remote excution job or ansible job.
++
+Possible values:
+// TODO: Values, numbers
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Job completed succssfully | OK | 0
+| Job not completed successfully | Error | 0
+|===
 
-Each sub-status can define its own set of possible values that are mapped to the three global status values.
+Compliance::
+Indicates if the host is compliant with OpenSCAP policies.
++
+Possible values:
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Compliant | OK | 0
+| Inconclusive | Warning | 1
+| At least one incompliant | Error | 2
+|===
 
-The *Build* sub-status has two possible values {endash} *pending* and *built* that are both mapped to global *OK* value.
+Inventory::
+Indicates if the host is synchronized to {RHCloud}.
+{ProjectServer} performs the this synchronization itself but only uploads basic information to {RHCloud}.
++
+Possible values:
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Disconnect | Warning | 0
+| Sync | OK | 1
+|===
 
-The *Configuration* status has more possible values that map to the global status as follows:
+Insights::
+Indicates if the host is synchronized to {RHCloud}.
+This synchronization is performend by the host.
+The host uploads more information than the {ProjectServer}.
++
+Possible values:
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Reporting | OK | 0
+| No report | Error | 1
+|===
 
-.Sub-statuses that map to the global *OK* status
+RHEL Lifecycle::
+Indicates the current state of the {RHEL} operating system installed of the host.
++
+Possible values:
```

Comment:
https://github.com/Katello/katello/blob/master/app/models/katello/rhel_lifecycle_status.rb

Reply:
Thank you for linking the file.
What what I am seeing, everything in this status should be mapped correctly.